Owncloud iOs Client/AppDelegate.m

```diff
@@ -1252,12 +1256,17 @@ - (void) restoreDownloadsInProccessFromDownloadFolderFromSystemWithIdentificator
     
     NSURLSessionConfiguration *configuration = nil;
     
-    if (IS_IOS8) {
-        configuration = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:identifier];
+    if (k_is_sso_active || !k_is_background_active) {
+        configuration = [NSURLSessionConfiguration defaultSessionConfiguration];
     } else {
-        configuration = [NSURLSessionConfiguration backgroundSessionConfiguration:identifier];
+        if (IS_IOS8) {
```

Comment:
We could remove this if and use for all cases ``` backgroundSessionConfigurationWithIdentifier: ``` instead of ```backgroundSessionConfiguration``` that is deprecated.